RunningOrder/fr.lproj/Localizable.strings

```diff
@@ -85,3 +86,12 @@
 
 "Label" = "Label";
 "Url" = "Url";
+
+// MARK: ToolbarItems
+
+"You can't undo this action." = "Vous ne pourrez pas annuler la suppression.";
+"Delete the story \"%@\" ?" = "Supprimer la story \"%@\" ?";
```

Comment:
same

Reply:
there is no alignment convention on this project for now. We might discuss it later :)